CMakeLists.txt

```diff
@@ -48,7 +46,7 @@ if (UPDATE_DEPS)
     message("********************************************************************************")
 
     set(optional_args)
-    if (NOT BUILD_TESTS)
```

Comment:
Is this actually a good things this variable is not name BUILT_TESTS? Isn't it inconsistent with other repository?

Reply:
It is inconsistent. But in this case I'd argue it's a good thing. It's generally good practice to group your cache variables together so it shows up nicely in cmake-gui.

![image](https://user-images.githubusercontent.com/114601453/220186317-aae08f7f-390a-45e9-b261-b7afdb1777de.png)

Reply:
Main reason I haven't fixed it in vvl is because it's already established.